conf/airframes/ENAC/cyfoam.xml

```diff
@@ -0,0 +1,307 @@
+    <!--<module name="gps" type="ubx_ucenter"/>-->
+    <module name="tlsf"/>
+    <module name="pprzlog">
+      <define name="SDLOG_START_DELAY" value="10"/>
+    </module>
+    <module name="logger" type="sd_chibios"/>
+    <!--<module name="flight_recorder"/>-->
+    <!--<module name="logger_spi_link"/>-->
+
+    <!--Use an airspeed sensor and get the measured airspeed in the messages-->
+    <module name="airspeed" type="ms45xx_i2c">
+      <define name="MS45XX_I2C_DEV" value="i2c1"/>
+      <define name="MS45XX_AIRSPEED_SCALE" value="2.4"/>
+      <define name="USE_AIRSPEED_MS45XX" value="TRUE"/>
+    </module>
+
+    <!--<module name="pwm_meas"/>-->
+    <!--<module name="AOA_pwm">-->
+      <!--<define name="AOA_PWM_CHANNEL" value="PWM_INPUT1"/>-->
+      <!--</module>-->
+
+    <module name="pwm_meas">
+      <define name="USE_PWM_INPUT1" value="PWM_PULSE_TYPE_ACTIVE_LOW"/>
+      <define name="USE_PWM_INPUT2" value="PWM_PULSE_TYPE_ACTIVE_LOW"/>
+    </module>
+  </modules>
+
+  <servos driver="Pwm">
+    <servo name="ELEVON_LEFT"  no="0" min="1000" neutral="1500" max="2000"/>
+    <servo name="ELEVON_RIGHT" no="1" min="1000" neutral="1500" max="2000"/>
+    <servo name="RM" no="2" min="1000" neutral="1100" max="2000"/>
+    <servo name="LM" no="3" min="1000" neutral="1100" max="2000"/>
+    <!--<servo name="SERVO_TEST" no="4" min="1000" neutral="1500" max="2000"/>-->
+  </servos>
+
+  <commands>
+    <axis name="PITCH"  failsafe_value="0"/>
+    <axis name="ROLL"   failsafe_value="0"/>
+    <axis name="YAW"    failsafe_value="0"/>
+    <axis name="THRUST" failsafe_value="0"/>
+  </commands>
+
+  <command_laws>
+    <set servo="ELEVON_LEFT"     value="autopilot.motors_on ? actuators_pprz[0] : 0"/>
+    <set servo="ELEVON_RIGHT"    value="autopilot.motors_on ? actuators_pprz[1] : 0"/>
+    <set servo="RM"              value="autopilot.motors_on ? actuators_pprz[2] : -MAX_PPRZ"/>
+    <set servo="LM"              value="autopilot.motors_on ? actuators_pprz[3] : -MAX_PPRZ"/>
+    <!--<set servo="RM"              value="autopilot_motors_on ? -MAX_PPRZ : -MAX_PPRZ"/>-->
+    <!--<set servo="LM"              value="autopilot_motors_on ? -MAX_PPRZ : -MAX_PPRZ"/>-->
+  </command_laws>
+
+  <section name="IMU" prefix="IMU_">
+    <!-- IMU mag calibration, make sure to calibrate the IMU properly before flight, see the wiki for more info-->
+    <!--Mag of gps-->
+    <define name="MAG_X_NEUTRAL" value="195"/>
+    <define name="MAG_Y_NEUTRAL" value="-47"/>
+    <define name="MAG_Z_NEUTRAL" value="-25"/>
+    <define name="MAG_X_SENS" value="3.68245686199" integer="16"/>
+    <define name="MAG_Y_SENS" value="3.81802043484" integer="16"/>
+    <define name="MAG_Z_SENS" value="4.01084591146" integer="16"/>
+
+    <define name="BODY_TO_IMU_PHI"   value="180." unit="deg"/>
+    <define name="BODY_TO_IMU_THETA" value="90." unit="deg"/>
+    <define name="BODY_TO_IMU_PSI"   value="180." unit="deg"/>
+  </section>
+
+  <section name="AUTOPILOT">
```

Comment:
This is usually at the bottom of the airframe. I think we should discuss what the preferred order is for the future... I think it is beneficial to have a standard but that standard hasn't been clearly defined. @gautierhattenberger is this defined anywhere?

Reply:
There is no definition of the order of the main nodes in airframe files at the moment. Even the DTD file is not reflecting anything in particular. Also the parser don't really care. We could edit somewhere a "recommended practice" guide. My preferred order is firmware, then "special nodes" (servos, commands, ...) and finally 'sections'.